### PR TITLE
Rewrite overview to add more use cases

### PR DESCRIPTION
**Pull request by marklewin**
## Description

I wanted to add more use cases and make the intro to this guide a bit more compelling, so that I didn't have to provide this info in the tutorials.

### CODE REVIEW

File: _documentation/voice/voice-api/guides/websockets.md [hunk 1]
```diff
@@ -8,9 +8,18 @@ navigation_weight: 7
 
 ## Overview
 
-The Nexmo Voice API allows you to connect to a call via a WebSocket. This means that you will have a real-time two-way stream of the raw audio in the call delivered to you over the WebSocket protocol. This allows you to connect the audio to platforms such as sentiment analysis, real-time transcription systems, and artificial intelligence.
+WebSockets is a computer communications [protocol](https://tools.ietf.org/html/rfc6455) that provides full-duplex communication channels over a single TCP connection.
```
Comment (mheap) [inline _documentation/voice/voice-api/guides/websockets.md]: This makes it sound more complicated than it is. I think I prefered the old phrasing

> you will have a real-time two-way stream of the raw audio in the call delivered to you over the WebSocket protocol

Comment (marklewin) [reply _documentation/voice/voice-api/guides/websockets.md]: I'm not sure this sentence makes sense. If it's a two-way stream, I'm not only getting the audio delivered to me, I'm sending it too?

I wanted to provide a definition of what a WebSocket is, before showing how it's of interest to VAPI developers. I've added a line to the second para that hopefully explains this a bit better.

Comment (marklewin) [reply _documentation/voice/voice-api/guides/websockets.md]: Oh, right, I see the extra sentence about playing audio into the call. But I still think that if we're talking about two-way audio we should specify how both directions work in the same sentence.

**Comment by tbedford** [comment]
I've merged in master to fix deployment error.

**Comment by tbedford** [comment]
@mheap I will leave you to merge if and when you are happy as you were original reviewer.

File: _documentation/voice/voice-api/guides/websockets.md (hunk 1 above)
Comment (mheap) [reply _documentation/voice/voice-api/guides/websockets.md]: It's the word `full-duplex` I think. Would most readers (including English as a second language) understand what it means? I think `two-way` is more accessible

Comment (marklewin) [reply _documentation/voice/voice-api/guides/websockets.md]: Agreed.

**Comment by mheap** [comment]
@marklewin Looks like our new spellchecker has it's first victim!

https://travis-ci.org/Nexmo/nexmo-developer/builds/553749521?utm_source=github_status&utm_medium=notification

```
1.68s$ ./node_modules/.bin/mdspell -r -n -a --en-us '_documentation/**/*.md' '_partials/*.md' '_partials/**/*.md' '_modals/**/*.md' '_tutorials/**.md'
    _documentation/voice/voice-api/guides/websockets.md
       17 | ng, transcribing or otherwise analysing calls using third party solut 
       17 | ng sentiment analysis in call centres to determine customer satisfa 
```

I think it wants `analyzing` and `call centers` (en-US) before it'll pass

**Comment by marklewin** [comment]
Nice! Fixed spelling.

**Pull request merged**